stacks-signer/src/runloop.rs

```diff
@@ -873,6 +1002,9 @@ impl From<&Config> for RunLoop<FireCoordinator<v2::Aggregator>> {
             signer_ids: config.signer_ids.clone(),
             signer_id: config.signer_id,
             signer_addresses: vec![],
+            nack_messages_received: HashMap::new(),
+            nack_messages_sent: HashSet::new(),
+            nack_threshold: get_nack_threshold(total_signers),
```

Comment:
Sorry to nit (again on this haha) but I think this should be a method of `RunLoop` so that it is always "tied" to the number of signers set elsewhere in the struct. It's just one less way of making a mistake in the future.

ie:

```rs
impl<C: Coordinator> RunLoop {
    // other code
    
    fn get_nack_threshold(&self) -> u32 {
        (&self.coordinator.get_config().num_signers + 1) / 2
    }
}
```

Reply:
This makes more sense!